lms/templates/dashboard.html

```diff
@@ -36,7 +36,10 @@
 
 <%block name="js_extra">
   <script src="${static.url('js/commerce/credit.js')}"></script>
-  <%static:js group='dashboard'/>
```

Comment:
Is there any benefit in grouping the static js? 
I know this is a legacy code but can we load these static js files with require js also?